scripts/third_party_size_check.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that runs a size check in third party folder and crashes if
+size limit is exceeded. The aim of this is to prevent us accidentally
+breaching the 10k file limit on App Engine.
+"""
+
+# pylint: disable=wrong-import-order
```

Comment:
Hmm, why disable?

Reply:
If I order imports like this:
```python
import sys
import os
```
I get this error:
```
----------------------------------------
ERROR: /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py Imports are incorrectly sorted.
--- /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py:before	2018-10-11 19:22:48.219414
+++ /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py:after	2018-10-11 19:24:16.355439
@@ -19,9 +19,8 @@
 breaching the 10k file limit on App Engine.
 """
 
+import os
 import sys
-import os
-
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000

----------------------------------------
```
And if I order like this:
```python
import os
import sys
```
I get this error:
```
----------------------------------------
ERROR: /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py Imports are incorrectly sorted.
--- /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py:before	2018-10-11 19:25:16.244338
+++ /home/tonadev/Documents/Work/OpenSource/oppia/scripts/third_party_size_check.py:after	2018-10-11 19:25:23.884525
@@ -21,7 +21,6 @@
 
 import os
 import sys
-
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000

----------------------------------------
```

Reply:
@tonadev: If you look closely at the diff, it asks to remove the one extra blank line which exists between the imports and  `THIRD_PARTY_PATH` constant. Try removing it and let me know if it works. Thanks!

Reply:
It works. Thanks!